assets/js/types/types.ts

```diff
@@ -149,4 +149,4 @@ export type Product = {
   imageUrl: string;
 };
 
-export type Card = "hidden" | "hide" | "share" | "chat" | "products";
+export type CardStatus = "hidden" | "share" | "chat" | "products";
```

Comment:
Shouldn't it be typescript [enum](https://www.typescriptlang.org/docs/handbook/enums.html)?